android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/BraveTabUiFeatureUtilities.java

```diff
@@ -17,12 +18,17 @@
 
     @SuppressLint("VisibleForTests")
     public static void maybeOverrideEnableTabGroupAutoCreationPreference(Context context) {
-        if (TabUiFeatureUtilities.isTabGroupsAndroidEnabled(context)
-                && TabUiFeatureUtilities.ENABLE_TAB_GROUP_AUTO_CREATION.getValue()) {
-            // Override it to make "Open in new tab" menu option in the context menu available.
+        if (TabUiFeatureUtilities.isTabGroupsAndroidEnabled(context)) {
+            // Override it to make "Open in new tab" menu option in the context menu available if
+            // applicable.
             SharedPreferencesManager.getInstance().writeBoolean(
-                    TAB_GROUP_AUTO_CREATION_PREFERENCE, false);
+                    TAB_GROUP_AUTO_CREATION_PREFERENCE, isBraveTabGroupsEnabled());
             CachedFeatureFlags.resetFlagsForTesting();
         }
     }
+
+    public static boolean isBraveTabGroupsEnabled() {
```

Comment:
nit: I guess it would be better if we use present context here, i.e. `isBraveTabGroupsEnable`

Reply:
It is present tense, just in passive form. It gets in line with Chromium's naming like `isMultiProcessEnabled`, `isCaptureEnabled` etc.